Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs

```diff
@@ -744,8 +750,10 @@ private void SpawnOperatives(int spawnCount, List<IPlayerSession> sessions, bool
 
             if (sessions.TryGetValue(i, out var session))
             {
-                var mob = _randomHumanoid.SpawnRandomHumanoid(_nukeopsRuleConfig.RandomHumanoidSettingsPrototype, _random.Pick(spawns), string.Empty);
                 var profile = _prefs.GetPreferences(session.UserId).SelectedCharacter as HumanoidCharacterProfile;
+                var mob = EntityManager.SpawnEntity(
+            _prototypeManager.Index<SpeciesPrototype>(profile?.Species ?? HumanoidAppearanceSystem.DefaultSpecies).Prototype,
+            _random.Pick(spawns));
```

Comment:
Can't profiles have deprecated species saved?

Reply:
This runs the code in the exact same way StationSpawningSystem does, so it should be fine(?).

Reply:
oh, that's definitely an issue